notebooks/peerscout/peerscout-create-view-editor-pubmed-ids.ipynb

```diff
@@ -0,0 +1,137 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "source": [
+    "project_id = 'elife-data-pipeline'\n",
+    "dataset = 'de_dev'\n",
+    "view_name='v_data_science_editor_pubmed_ids'\n",
+    "mview_name='mv_data_science_editor_pubmed_ids'\n",
+    "query_name ='v_data_science_editor_pubmed_ids.sql'"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "source": [
+    "import logging\n",
+    "\n",
+    "from data_science_pipeline.utils.bq import get_client\n",
+    "from data_science_pipeline.sql import get_sql\n",
+    "from bigquery_views_manager.update_views import update_or_create_view\n",
+    "from bigquery_views_manager.materialize_views import materialize_view\n"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "source": [
+    "LOGGER = logging.getLogger(__name__)\n",
+    "logging.basicConfig(level='INFO')"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "source": [
+    "client = get_client(project_id=project_id)"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "source": [
+    "LOGGER.info('processing: %s', query_name)\n",
+    "view_query = get_sql(query_name).format(\n",
+    "        project=project_id,\n",
+    "        dataset=dataset\n",
+    ")\n",
+    "update_or_create_view(\n",
+    "        client=client,\n",
+    "        view_name=view_name,\n",
+    "        view_query=view_query,\n",
+    "        dataset=dataset\n",
+    ")    "
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
```

Comment:
I'd remove outputs from this notebook too. The nbnotebook strip should do that.
We could look into getting the pre-commit hook for for you.

In any case it would be good to make the linting to check for outputs functional again. Perhaps you could create an issue for it? (It should in general work as long as Jupyter is used via the container. For it to be more flexible the linting can be amended)